mw-webapp/src/component/tooltip/positionTooltip/PositionTooltip.ts

```diff
@@ -0,0 +1,6 @@
+export const enum PositionTooltip {
```

Comment:
1. please add some documentation to all public members
2. PositionTooltip.TOP is even more transparent than PositionTooltip.TOOLTIP_TOP. Get rid of useless words

Reply:
1. Done
2. Done